openfisca_france/reforms/plf2015.py

```diff
@@ -26,23 +26,24 @@
 from __future__ import division
 
 from openfisca_core import formulas, periods, reforms
-
+from ..model.base import *
 from ..model.prelevements_obligatoires.impot_revenu import ir
 
 
 def build_reform(tax_benefit_system):
     Reform = reforms.make_reform(
         key = 'plf2015',
-        name = u'Projet de Loi de Finances 2015 appliquée au revenus 2013',
+        name = u'Projet de Loi de Finances 2015 appliquée aux revenus 2013',
         reference = tax_benefit_system,
         )
 
     @Reform.formula
-    class decote(formulas.SimpleFormulaColumn):
+    class decote(formulas.DatedFormulaColumn):
         label = u"Nouvelle décote 2015"
         reference = ir.decote
 
-        def function(self, simulation, period):
+        @dated_function(start = date(2013, 1, 1), stop = date(2013, 12, 31) )
+        def function_2013(self, simulation, period):
```

Comment:
It is intentionnal. You can change the label to Nouvelle décote 2015 sur revenus 2013.
When PLF2015 (revenus 2014) is announced we have only the 2013 legislation so we do a reform on that year.
When we have the full 2014 legislation we implement it and we shoud also implement a 2014 business as usual (not done yet) so you can plau your reform on 2013 and 2014.